{{cookiecutter.project_slug}}/config/settings/production.py

```diff
@@ -187,69 +192,20 @@ class MediaRootS3Boto3Storage(S3Boto3Storage):
 
 {% endif %}
 {%- if cookiecutter.use_sentry == 'y' -%}
-# raven
-# ------------------------------------------------------------------------------
-# https://docs.sentry.io/clients/python/integrations/django/
-INSTALLED_APPS += ['raven.contrib.django.raven_compat']  # noqa F405
-MIDDLEWARE = ['raven.contrib.django.raven_compat.middleware.SentryResponseErrorIdMiddleware'] + MIDDLEWARE
-
 # Sentry
 # ------------------------------------------------------------------------------
 SENTRY_DSN = env('SENTRY_DSN')
-SENTRY_CLIENT = env('DJANGO_SENTRY_CLIENT', default='raven.contrib.django.raven_compat.DjangoClient')
-LOGGING = {
-    'version': 1,
-    'disable_existing_loggers': True,
-    'root': {
-        'level': 'WARNING',
-        'handlers': ['sentry'],
-    },
-    'formatters': {
-        'verbose': {
-            'format': '%(levelname)s %(asctime)s %(module)s '
-                      '%(process)d %(thread)d %(message)s'
-        },
-    },
-    'handlers': {
-        'sentry': {
-            'level': 'ERROR',
-            'class': 'raven.contrib.django.raven_compat.handlers.SentryHandler',
```

Comment:
Looks like we can re-use this config and just change the handler here to the [new `SentryHandler`](https://github.com/getsentry/sentry-python/blob/18cd8bf056fbee13456140cc2fc3059b8b973f3e/sentry_sdk/integrations/logging.py#L47)? I'm not too sure to understand how the logging integration works with other loggers TBH.

Reply:
This is all no longer necessary
From the docs
> Most SDKs will now automatically collect data if available

Reply:
I would be in favor on placing some sort of comment regarding on WHY logging configuration is not longer necessary, specially for people not very familiar with django's default config.